packages/indexer-prisma/prisma/schema.prisma

```diff
@@ -168,11 +168,56 @@ model username_claimed {
 }
 
 model shares_token_user {
-  id            String   @id
-  owner         String
-  token_address String
-  amount_owned  Decimal? @default(0) @db.Decimal
-  is_claimable  Boolean? @default(false)
-  amount_claimed Decimal? @default(0) @db.Decimal
+  id             String    @id
+  owner          String
+  token_address  String
+  amount_owned   Decimal?  @default(0) @db.Decimal
+  is_claimable   Boolean?  @default(false)
+  amount_claimed Decimal?  @default(0) @db.Decimal
+  created_at     DateTime? @default(now()) @db.Timestamp(6)
+}
+
+model tip_deposit {
+  network          String?
+  block_hash       String?
+  block_number     BigInt?
+  block_timestamp  DateTime? @db.Timestamp(6)
+  transaction_hash String?
+  deposit_id       String    @id
+  sender           String
+  nostr_recipient  String
+  amount           Decimal?  @db.Decimal
+  token_address    String
+  created_at       DateTime? @default(now()) @db.Timestamp(6)
```

Comment:
we can add here few attributes to update:

is_cancel: boolean
is_claimed: boolean
gas_amount: number

We can update this row table of the deposit when someone cancel or claim a deposit nostr